plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/common/core/server/ServerUtils.java

```diff
@@ -68,7 +68,7 @@ public static void setProjectAttribute(String name, String value, String nodeQua
 		node.put(name, value);
 		if( sync ) {
 			try {
-				node.sync();
+				node.flush();
```

Comment:
the method parameter [boolean sync] effectively does a flush and I believe should therefore be called accordingly [boolean flush] (since sync is an operation that's available on nodes, too)